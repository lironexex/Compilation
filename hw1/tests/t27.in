// comments 7 - double closing patterns for multi-line comment
/* some comment 
written here */ */
