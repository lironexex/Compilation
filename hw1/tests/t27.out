1 COMMENT 1
3 COMMENT 2
3 BINOP *
3 BINOP /
